lib/dal/tests/integration_test/schema/variant/authoring/update_variant.rs

```diff
@@ -0,0 +1,130 @@
+use dal::diagram::Diagram;
+use dal::prop::PropPath;
+use dal::schema::variant::authoring::VariantAuthoringClient;
+use dal::{ChangeSet, DalContext, Prop};
+use dal_test::helpers::create_component_for_schema_name;
+use dal_test::test;
+
+#[test]
+async fn update_variant(ctx: &mut DalContext) {
+    let new_change_set = ChangeSet::fork_head(ctx, "new change set")
+        .await
+        .expect("could not create new change set");
+    ctx.update_visibility_and_snapshot_to_visibility(new_change_set.id)
+        .await
+        .expect("could not update visibility");
```

Comment:
You get one for free when you start the test! Not sure if you wanted another one though.

Reply:
ah great! Removing this then :)